content/api/javascript/apireference.md

```diff
@@ -322,24 +318,31 @@ appbaseRef
 
 -   **params** `Object` <br>A JavaScript object containing the query `type` and `body`.
 
-        	- **type** ``String`` <br>Document type
-        	- **body** ``Array`` <br>An array specifying search requests in header followed by body order for each request.
+    -   **type** `String` <br>Document type
+    -   **body** `Array` <br>An array specifying search requests in header followed by body order for each request.
 
 **Returns**
 Promise.
 
 ### reactiveSearchv3()
+
 ReactiveSearch method allows you to execute the search requests securely with the help of newly launched `ReactiveSearch` API. You can read about `ReactiveSearch` API [here](/docs/search/reactivesearch-api/).
+
 ```js
 appbaseRef
-	.reactiveSearchv3([{
-		id: "book_search",
-		dataField: ["original_title"],
-		size: 10,
-		value: "harry"
-	}], {
-		userId: "jon@appbase.io"
-	})
+	.reactiveSearchv3(
```

Comment:
@lakhansamani This looks like an unintended side-effect.

Reply:
@siddharthlatest it was unintended earlier, it looks correct now. Verified in preview.